### DISCUSSION TITLE
Game crashing randomly with few mods

### DISCUSSION THREAD

**Discussion by P26601** [FabricMC/fabric-api#3617] [Player support]
### Minecraft Logs

```txt
https://pastebin.com/FL2M2eFM
```

### Description

Game seems to crash randomly when exploring. Save can be re-loaded but game crashes again after a few minutes, error log shows nothing obvious (at least for me lol). Anyone got an idea what might cause the crashes?

**Comment by Poopooracoocoo** [comment]
Just spitballing. The [MixinTrace](https://modrinth.com/mod/mixintrace) mod might make your crash report more useful. You could also look at your latest.log from your logs folder.

**Reply by P26601** [reply]
Thanks! [This](https://pastebin.com/ZtrsMcEJ) is the new crash report with MixinTrace. Seems like there's an issue with Fabric itself or Iris?
